moveit_configs_utils/src/moveit_configs_utils/moveit_configs_builder.py

```diff
@@ -0,0 +1,201 @@
+from pathlib import Path
+from typing import Optional, List
+import logging
+from ament_index_python.packages import get_package_share_directory
+
+from parameter_builder import ParameterBuilder, load_yaml, load_xacro
+
+
+class MoveItConfigs(object):
+    def __setattr__(self, name, value):
+        if hasattr(self, name):
+            object.__setattr__(self, name, value)
+        else:
+            raise AttributeError(
+                f"Cannot set name {name} on object of type {self.__class__.__name__}"
+            )
+
+    robot_description = None
+    robot_description_semantic = None
+    robot_description_planning = None
+    robot_description_kinematics = None
+    planning_pipelines = None
+    trajectory_execution = None
+    planning_scene_monitor = None
+    move_group_capabilities = None
+    move_group = None
+    joint_limits = None
```

Comment:
Some planners (at least the PILZ planner) require Cartesian limits which look i.e. like this:
```
cartesian_limits:
  max_trans_vel: 1
  max_trans_acc: 2.25
  max_trans_dec: -5
  max_rot_vel: 1.57
```
Maybe you could add this as a default field too.

Reply:
Done